pkgs/by-name/we/werf/package.nix

```diff
@@ -103,7 +103,7 @@ buildGoModule (finalAttrs: {
       Buildah.
     '';
     homepage = "https://werf.io";
-    changelog = "https://github.com/werf/werf/releases/tag/${finalAttrs.src.rev}";
+    changelog = "https://github.com/werf/werf/releases/tag/${finalAttrs.src.tag}";
```

Comment:
```suggestion
    changelog = "https://github.com/werf/werf/releases/tag/v${finalAttrs.version}";
```

Reply:
I don't think that makes sense in here. The changelog is only accessible via GitHub releases on a specific tag. If a user overrides a version to something custom this link won't be useful anyway.

Reply:
I think in most cases it makes sense to use `finalAttrs.version` here (which is also what most other packages do, afaik) because you would usually override the version to a different (either newer or older) release. In that case I would expect meta.changelog to be updated accordingly. If for whatever reason you really need to change the version to something completely different, you can just override meta.changelog as well. If we use finalAttrs.src.tag here, the url could either be outdated or could fail to evaluate if src doesn't have a tag.

Reply:
Alright.